filebeat/input/v2/input-cursor/input.go

```diff
@@ -147,6 +147,7 @@ func (inp *managedInput) runSource(
 	}()
 
 	client, err := pipeline.ConnectWith(beat.ClientConfig{
+		InputID:       ctx.ID,
```

Comment:
I think this should use `IDWithoutName` instead of `ID`.  `IDWithoutName` is what tracks to the `id` given in the config. The `ID` appends extra info such as the file path or URL.

Reply:
Yeah, it's the case for the [cel input](https://github.com/elastic/beats/pull/42618/files#diff-5e5ffd05859173510bcc2768df4ea1ae999e5528baf5e3165f1416f9f9e127fcR115) as well. 

SO, can I let it as is?